src/libraries/System.Text.Json/tests/Serialization/ContinuationTests.NullToken.cs

```diff
@@ -0,0 +1,148 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    public static partial class ContinuationTests
+    {
+        // This validates the original issue is fixed against the original
+        // test data (search for "CustomerCollectionResponse" to get
+        // the details - when github still exists you may have luck)
```

Comment:
nit: Your original comment which linked to the issue was a sufficient and succinct way to give the reader a pointer to follow for more info.
```suggestion
        // From https://github.com/dotnet/runtime/issues/42070
```

Reply:
+1 please return to what you had previously.